### ISSUE TITLE
runtime error in VdCnn.cs

### ISSUE THREAD

**Issue by Hondo2019** [SciSharp/TensorFlow.NET#328]
Just playing with your VdCnn sample, which throws excpetion 'Unable to find an entry point namend 'AddControlInput' in DLL tensorflow. Happens on that line:

                       // batch normalization
                        conv = tf.layers.batch_normalization(conv, training: is_training);

The references are SciSharp.TensorFlow.Redist v1.14.0 and TensorFlow.NET v0.10.3

Thanks for your help!

**Comment by Oceania2018** [comment]
Have to replace `tensorflow` with this one: https://github.com/SciSharp/TensorFlow.NET/tree/master/tensorflowlib/runtimes/win-x64/native

**Comment by Hondo2019** [comment]
Hi! I just upgraded to your latest TensorFlow.NET 0.10.10 and used the dll you provided.

Now I get 'can't get graident function through get_gradient_function TopKV2' on this line:

               tf_with(tf.control_dependencies(update_ops.Select(x => (Operation)x).ToArray()), delegate
                {
                    var adam = tf.train.AdamOptimizer(learning_rate);
                    adam.minimize(loss, global_step: global_step);
                });

**Comment by Hondo2019** [comment]
Stack trace:

System.NotImplementedException: can't get graident function through get_gradient_function TopKV2
   at Tensorflow.ops.get_gradient_function(Operation op)
   at Tensorflow.gradients_util.<>c__DisplayClass0_0.<_GradientsHelper>b__0(NameScope scope)
   at Tensorflow.Python.tf_with[T](T py, Action`1 action)
